docs_src/microservices/device/services/device-usb-camera/walkthrough/custom-build.md

```diff
@@ -82,13 +82,29 @@ See the examples at `cmd/res/devices`
 !!! Note 
     When a new device is created in Core Metadata, a callback function of the device service will be called to add the device card name and serial number to protocol properties for identification purposes. These two pieces of information are obtained through `V4L2` API and `udev` utility.
 
-## Configurable RTSP server hostname and port
-Enable/Disable RTSP server and set hostname and port in the `Driver` section of `device-usb-camera/cmd/res/configuration.yaml` file. The default values can be used in this guide. The RtspAuthenticationServer value indicates the internal hostname and port on which the device service will listen for RTSP authentication requests on. If this value is changed, you will have to also change the [mediamtx configuration](../supplementary-info/advanced-options.md#authentication-server-configuration) to point to the new hostname/port as well.
+## Configure RTSP server mode, hostname, and port
+Set the RTSP server hostname and port in the `Driver` section of `device-usb-camera/cmd/res/configuration.yaml` file.
```

Comment:
`device-usb-camera/cmd/res/configuration.yaml` should point to the actual location. In most of the places in docs thats how it is.